tests/src/utils/test_eth_transaction.py

```diff
@@ -49,7 +49,7 @@ async def test_should_decode_all_transactions_types(
             assert expected_to == decoded_tx["destination"]
             assert transaction["value"] == int(decoded_tx["amount"], 16)
             # pre-eip155 txs have an internal chain_id set to 0 in the decoded tx
-            assert transaction.get("chainId", 0) == decoded_tx["chain_id"]
+            assert transaction.get("chainId", 0) == decoded_tx["chain_id"]["value"]
```

Comment:
I4d rather update `serde.py` to serialize the chain_id as None or value directly